src/App.tsx

```diff
@@ -1,41 +1,100 @@
-import React from 'react';
-import './App.scss';
-import './styles/general.scss';
+import React, { useState, useEffect } from 'react';
+
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getUsers } from './api/users';
+import { getUserPosts } from './api/posts';
+
+import './App.scss';
+import './styles/general.scss';
+
+const App: React.FC = () => {
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [users, setUsers] = useState<User[]>([]);
+  const [postId, setPostId] = useState(0);
+  const [userId, setUserId] = useState(0);
+  const [loadPost, setLoadPost] = useState(true);
+  const [toggleComments, setToggleComments] = useState(false);
```

Comment:
it will be good to store this state in post details, where comments are rendered